### PR TITLE
Added common_bundle() function for OSX

### PR DESCRIPTION
**Pull request by rdumusc**

### CODE REVIEW

**Comment by eile** [comment]
Why not merge bundle into application, e.g., as an option (or even per default?)

**Comment by rdumusc** [comment]
Actually not a bad idea! Not by default (generally common_application() should do a regular command line application) but we could apply the extra steps if MACOSX_BUNDLE was passed in ARGV. Don't merge yet, I will prepare a new patch for that.

**Comment by eile** [comment]
That's exactly the question: Is command line or UI application the "default"? Shall we make it dependent on the presence of Qt?

**Comment by rdumusc** [comment]
updated

File: CommonApplication.cmake [hunk 1]
```diff
@@ -13,6 +13,13 @@
 # ** NAME_UI_FORMS list of all .ui input files
 # ** NAME_RESOURCES list of all .qrc resource files
 #
+# Cross-plateform wrapper around common_application() for GUI applications:
```
Comment (eile) [inline CommonApplication.cmake]: Frypo: Platform

**Comment by eile** [comment]
+1

File: CommonApplication.cmake (hunk 1 above)
Comment (rdumusc) [inline CommonApplication.cmake]: Thanks, I keep doing this frypo :-)

**Pull request merged**